addon/components/on-this-page.hbs

```diff
@@ -4,9 +4,11 @@
     <hr>
     <ul>
       {{#each @toc as |toc|}}
-        <li>
-          <a href="#{{toc.id}}">{{toc.text}}</a>
-        </li>
+       {{#if (or (eq toc.depth '1') (eq toc.depth '2') (eq toc.depth '3'))}}
```

Comment:
This is pretty verbose, but keeps the component a TOC. Depth is passed as string, and changing that to a number would have a bigger impact and be outside of scope of this repo?

Reply:
I'm happy with it as it is 👍 if you wanted to make this a helper like `(should-show-toc-depth)` then you could absorb the string/number issue there, but that's not necessary 👍